v4l2/v4l2_encode.cpp

```diff
@@ -58,6 +58,14 @@ bool V4l2Encoder::start()
     streamFormat.streamFormat = AVC_STREAM_FORMAT_ANNEXB;
     m_encoder->setParameters(VideoConfigTypeAVCStreamFormat, &streamFormat);
 
+    if (m_svct) {
+        VideoTemporalLayers layers;
+        layers.numLayersMinus1 = 1; // It is two layers actually
+        layers.bitRate[0] = 3000000;
```

Comment:
This hard coded bitrate may introduce issue on dynamic bitrate changes. Could we change logical like this:
if total birate is a, the zero layer bitrate is a/2 etc.